### ISSUE TITLE
Geneve tunnel support 

### ISSUE THREAD

**Issue by yossi-r** [F5Networks/f5-declarative-onboarding#206]
Is your feature request related to a problem? Please describe.
I need to be able to declare a Geneve Tunnel configuration.
I need to be able to declare a self-ip that listens on the Geneve tunnel

Describe the solution you'd like

I would like to be able to include declarations for a Geneve tunnel and reference it in Virtual Server, Self-IP and Route declarations, as follows:

"geneveTunnel1": {
    "class": "Tunnel",
    "type": "geneve",
    "localAddr": "10.90.4.8",
    "mode": "bidirectional",
    "mtu": "0",
    "usePMTU": true,
    "isTransparent": false
},
"geneveTunnel1-self": {
   "class": "SelfIp",
   "address": "10.90.4.8/24",
   "vlan": "geneveTunnel1",
   "allowService": "default",
   "trafficGroup": "traffic-group-local-only"
}

Describe alternatives you've considered

scripting tmsh commands :(

Additional context

**Comment by grf5** [comment]
It appears that much of this was enabled in #98 but we need the following attributes added to the tunnel object:

- local-address
- remote-address

Sample configuration:

```
net tunnels tunnel geneve {
    app-service none
    auto-lasthop default
    description none
    idle-timeout 300
    if-index 128
    key 0
    local-address 10.250.250.173%1
    mode bidirectional
    mtu 0
    partition Common
    profile geneve
    remote-address any%1
    secondary-address any6
    tos preserve
    traffic-group traffic-group-local-only
    transparent disabled
    use-pmtu enabled
}
net self geneve-tunnel {
    address 10.131.0.1%1/24
    address-source from-user
    allow-service all
    app-service none
    description none
    floating disabled
    inherited-traffic-group false
    partition Common
    service-policy none
    traffic-group traffic-group-local-only
    unit 0
    vlan geneve
}
```

**Comment by dstokesf5** [comment]
Support for configuring Geneve tunnels will be available in the  1.24.0 release.

**Comment by dstokesf5** [comment]
This was resolved with the 1.24.0 release of Declarative Onboarding.
